tests/integration/techRecords.intTest.ts

```diff
@@ -290,6 +293,237 @@ describe("techRecords", () => {
       });
     });
   });
-});
 
+  describe("postTechRecords", () => {
+    context("when database is populated", () => {
+      beforeAll(async () => {
+        await emptyDatabase();
+      });
+
+      afterAll(async () => {
+        await populateDatabase();
+      });
+
+      context("when database is populated", () => {
+        beforeEach(async () => {
+          await populateDatabase();
+        });
+
+        afterEach(async () => {
+          await emptyDatabase();
+        });
+
+        // @ts-ignore
+        const newTechRec: ITechRecordWrapper = mockData[0];
```

Comment:
this is a data mutation which may have consequences for future tests. Please use `{...mockData[0]}` to get a shallow copy of the object (shallow is good enough for data from a flat file)